src/resources/cubemap.js

```diff
@@ -120,11 +120,7 @@ class CubemapHandler {
                     resources[i + 1] = new Texture(this._device, {
                         name: cubemapAsset.name + '_prelitCubemap' + (tex.width >> i),
                         cubemap: true,
-                        // we have some old assets that were not created in RGBM format (see Gangnam
-                        // project). These assets appear to have predated the texture type member. So
-                        // here we assume assets with a texture type member were created in RGBM and
-                        // older assets were created without.
-                        type: getType() ? TEXTURETYPE_RGBM : TEXTURETYPE_DEFAULT,
+                        type: getType() || tex.type,
```

Comment:
do you want to keep some new comment here?

Reply:
done!